src/psyclone/tests/gocean1p0_transformations_test.py

```diff
@@ -1475,7 +1475,8 @@ def test_ocl_apply(kernel_outputdir):
     # transformation to something that is not an InvokeSchedule
     with pytest.raises(TransformationError) as err:
         _, _ = ocl.apply(schedule.children[0])
-    assert "the supplied node must be a (sub-class of) Schedule " in str(err)
+    assert "the supplied node must be a (sub-class of) InvokeSchedule " \
+        "" in str(err)
```

Comment:
I don't think you need the '""' on this line.

Reply:
Ups, deleted.